django/template/base.py

```diff
@@ -197,6 +197,9 @@ def compile_nodelist(self):
         try:
             return parser.parse()
         except Exception as e:
+            if self.origin.name != UNKNOWN_SOURCE:
+                template_path = "Template : %s, " % self.origin.name
```

Comment:
Small nitpick, can you remove the space? `Template:`

Reply:
Sure thing.  Catching up and fixing tests. Update to PR forthcoming.